src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -159,7 +164,11 @@ public SeekableByteChannelPrefetcher(SeekableByteChannel chan, int bufSize) thro
             this.bufSize = bufSize;
         }
         this.open = true;
-        exec = Executors.newFixedThreadPool(1);
+        // make sure the prefetching thread's name indicate what it is and
+        // which prefetcher it belongs to (for debugging purposes only, naturally).
+        String nameFormat = "nio-prefetcher-" + prefetcherIndex + "-thread-%d";
+        ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat(nameFormat).build();
```

Comment:
This is a separate issue, but I think we should make the prefetcher threads into daemon threads.  
We're seeing an issue where the main thread dies with an exception but the prefetcher threads stay active and prevent the program from closing.  Making them daemons should cause them to die appropriately if the main thread dies I believe.